manifests/0000_10_config-operator_01_consoleclidownload.crd.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Rebase, and I have to say I like the output of this much better.  It's limited to the fields that specifically matter to the custom resource.